R/bcdc_options.R

```diff
@@ -101,8 +100,31 @@ bcdc_single_download_limit <- function() {
     ))
 
     res <- cc$parse("UTF-8")
-    doc <- xml2::read_xml(res)
+    xml2::read_xml(res)
+
+  } else {
+    message("No access to internet")
+    invisible(FALSE)
```

Comment:
Should this return `NULL` instead of `FALSE` if no internet? Then the functions that use it can check if `is.null()` and if so return/fail early?

Reply:
Yes absolutely